api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -232,7 +232,7 @@ public void testSimulateUserFlowThroughRenewal() {
         .isEqualTo(TierAccessStatus.ENABLED);
 
     // Simulate time passing, user is no longer compliant
-    PROVIDED_CLOCK.setInstant(START_INSTANT.plus(1, ChronoUnit.DAYS));
+    PROVIDED_CLOCK.setInstant(START_INSTANT.plus(2, ChronoUnit.DAYS));
```

Comment:
I believe I modified the logic here slightly in that now the module won't be considered expired until *after* the expiration date, which I think is fine.